src/index.html

```diff
@@ -56,6 +56,8 @@ <h1>Watch the Video</h1>
     </div>
     </main>
     <script src="script.js"></script>
-    
+    <footer>
+      Copyright &copy; Full Stack Web Dev
```

Comment:
Usually when we have symbols we use the HTML entities / code like what you saw here.

https://www.w3schools.com/html/html_entities.asp

Out of curiosity if you really wanted to show a less-than sign, you would have to write &lt;

See the pattern? Always ampersand, abbreviation (`lt`, less than) and semicolon.